src/Ribbon.js

```diff
@@ -6,26 +6,77 @@ import axios from 'axios';
 import RibbonBase from './RibbonBase';
 import AssociationsView from './view/AssociationsView';
 import SpeciesLabel from './view/SpeciesLabel';
+// import FaIconPack from 'react-icons/lib/fa'
+// import FaClose from 'react-icons/lib/fa/close'
+import * as FontAwesome from 'react-icons/lib/fa'
```

Comment:
If at all possible, please import only modules as needed (instead of *) to keep bundled asset to minimal size.

Reply:
Change made!    Much cleaner to look at that way.   Had never seen this library.  Really like it.